coalib/settings/ConfigurationGathering.py

```diff
@@ -209,6 +209,13 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     log_printer.log_level = LOG_LEVEL.str_dict.get(str_log_level,
                                                    LOG_LEVEL.INFO)
 
+    if sections['cli'].contents and 'default' in sections:
+        logging.warning('Inheritance of CLI arguments to \'Default\' section '
+                        'has been deprecated. Please modify the \'Default\' '
+                        'section by using the --settings argument.')
```

Comment:
I don't feel this message is good, as a user I'm not supposed to use normal cli args anymore? I'm not supposed to use the default section anymore right? And if I do that my cli settings won't be stored, is that correct?